ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/cli/Cli.scala

```diff
@@ -28,6 +28,21 @@ object Cli {
     override val reads: String => Duration = Duration.parse
   }
 
+  private implicit val levelRead: Read[Level] = new Read[Level] {
+    override def arity: Int = 1
+    override val reads: String => Level = (logLevel: String) => {
+      // format: off
+      logLevel match {
+        case "INFO"  | "info"  => Level.INFO
+        case "TRACE" | "trace" => Level.TRACE
+        case "DEBUG" | "debug" => Level.DEBUG
+        case "WARN"  | "warn"  => Level.WARN
+        case "ERROR" | "error" => Level.ERROR
```

Comment:
what happens when it's none of these?

Reply:
minor comment on formatting: I very much agree with this style, but this could be configured as the style scalafmt formats to, which we used to have before open-sourcing. I find it silly to disable formatting just so we can achieve what we had before **with** formatting. cc @francesco-da

Reply:
Added an explicit handling of this case. Previously it would be handled implicitly by scopt with a less-than-helpful error message.

Reply:
@gerolf-da please mind my comment on formatting above